### PR TITLE
Remove duplicate comment words

### PR DESCRIPTION
**Pull request by dstadulis**
Duplicate words are identified with the regex:

^[\s\t]*//.*\b(\w+)\b \1\b
^[\s\t]*#.*\b(\w+)\b \1\b

and removed

### CODE REVIEW

File: tapdb/interfaces.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ type TxOptions interface {
 type BatchedTx[Q any] interface {
 	// ExecTx will execute the passed txBody, operating upon generic
 	// parameter Q (usually a storage interface) in a single transaction.
-	// The set of TxOptions are passed in in order to allow the caller to
+	// The set of TxOptions are passed in order to allow the caller to
```
Comment (guggero) [inline tapdb/interfaces.go:52]: I think this one is actually on purpose. Could perhaps be written as "passed in in-order" to be more clear.

Comment (GeorgeTsagk) [reply tapdb/interfaces.go:52]: "in-order" as in ordered in a specific way? after looking at `TxOptions` there doesn't seem to be any slice or array that holds entities in some order. So perhaps change the phrasing completely and do something like
```suggestion
	// The set of TxOptions are provided in order to allow the caller to
```

**Review by GeorgeTsagk — APPROVED** [review]
LGTM, pending the "in in order" nit

File: tapdb/interfaces.go (hunk 1 above)
Comment (guggero) [reply tapdb/interfaces.go:52]: Ah, no, I misread it myself. So your suggestion is actually better.

**Comment by guggero** [comment]
!lightning-labs-deploy mute

**Comment by guggero** [comment]
@dstadulis are you ever going to update this PR?

**Pull request closed without merging**